app/src/main/res/values/strings.xml

```diff
@@ -175,6 +175,7 @@
     <string name="guardian_tooMuchMoney">Congratulations, you seem to have a nice stack of Bitcoin!\nPlease keep in mind that Zap is a Hot Wallet.\nWe do our best to keep this wallet secure, but for your own security sake, we recommend you consider Cold Wallets or Hardware Wallets to store your major savings.</string>
     <string name="guardian_notReadyForMainnet">You are on MAINNET with real Bitcoin!\nZap Android is in early development phase. You might lose funds!\n\nIf you don\'t want to take that risk, switch to testnet in the meantime.</string>
     <string name="guardian_remoteConnect">Are you sure you want to connect to \n\n%1$s\n\nPlease check the hostname carefully.</string>
+    <string name="guardian_blockExplorer">By looking up information with a Block Explorer your IP-Address gets exposed. In most cases this is no problem, but if you want to stay 100%% anonymous, you should either not use it or go to the settings and choose a Block Explorer that supports tor.</string>
```

Comment:
typo: `100%%` - maybe rephrase like `if you want to increase your anonymity`.  Would not suggest the user that there is 100% anonymity possible.  Would capitalize **T**or

Reply:
100%% is actually correct, if you only have 100% it will cause a lint error. % is escaping
But I like your suggestion, so it will be removed anyway.